src/md/community.md

```diff
@@ -4,9 +4,9 @@
 
 The following mailing lists are available for Apex.
 
-- [dev@apex.incubator.apache.org](http://mail-archives.apache.org/mod_mbox/incubator-apex-dev/) is for people who want to contribute code to Apex. [subscribe](mailto:dev-subscribe@apex.incubator.apache.org?subject=send%20this%20email%20to%20subscribe), [unsubscribe](mailto:dev-unsubscribe@apex.incubator.apache.org?subject=send%20this%20email%20to%20unsubscribe), [archives](http://mail-archives.apache.org/mod_mbox/incubator-apex-dev/)
-- [users@apex.incubator.apache.org](http://mail-archives.apache.org/mod_mbox/incubator-apex-users/) is a discussion forum for people who want to build apps and/or solutions using Apex. [subscribe](mailto:users-subscribe@apex.incubator.apache.org?subject=send%20this%20email%20to%20subscribe), [unsubscribe](mailto:users-unsubscribe@apex.incubator.apache.org?subject=send%20this%20email%20to%20unsubscribe), [archives](http://mail-archives.apache.org/mod_mbox/incubator-apex-users/)
-- [commits@apex.incubator.apache.org](http://mail-archives.apache.org/mod_mbox/incubator-apex-commits/) is for commit messages and patches to Apex. [subscribe](mailto:commits-subscribe@apex.incubator.apache.org?subject=send%20this%20email%20to%20subscribe), [unsubscribe](mailto:commits-unsubscribe@apex.incubator.apache.org?subject=send%20this%20email%20to%20unsubscribe), [archives](http://mail-archives.apache.org/mod_mbox/incubator-apex-commits/)
+- [dev@apex.incubator.apache.org](http://mail-archives.apache.org/mod_mbox/incubator-apex-dev/) is for people who want to contribute code to Apex. [subscribe](mailto:dev-subscribe@apex.incubator.apache.org?subject=send%20this%20email%20to%20subscribe), [unsubscribe](mailto:dev-unsubscribe@apex.incubator.apache.org?subject=send%20this%20email%20to%20unsubscribe), [forum](https://s.apache.org/apex-dev), [archives](http://mail-archives.apache.org/mod_mbox/incubator-apex-dev/)
```

Comment:
See comment below. How does that work? Communication needs to be on the mailing list.